### DISCUSSION TITLE
The version of axel tool is out of date

### DISCUSSION THREAD

**Discussion by draftman9** [ScoopInstaller/Scoop#5489] [General]
The latest version of axel is 2.16.1-1 in scoop. But the latest version should be 2.17 and the 2.16.1-1 has some glitches now. you can find out [here](https://github.com/axel-download-accelerator/axel/issues/404)

**Comment by rashil2000** [comment]
I have deprecated the axel manifest: https://github.com/ScoopInstaller/Main/commit/af68f2fe68dfa58fe8ab3657e10805e4e5a0c154
